### PR TITLE
Test components

### PR DESCRIPTION
**Pull request by Oxyrus**
Adds unit tests for all the components we've worked on except for `CreateIdea`, that one is causing trouble, let me know if anyone can get a test working for it.

### CODE REVIEW

File: client/src/components/ideas/CreateIdea.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
         <div class="create__form-tags">
           <div class="create__tag-wrap">
-            <span class="create__form-tag" v-for="(tag, index) in tags">
+            <span class="create__form-tag" v-for="(tag, index) in tags" v-bind:key="index">
```
Comment (Trion129) [inline client/src/components/ideas/CreateIdea.vue]: Why this change?

Comment (Oxyrus) [reply client/src/components/ideas/CreateIdea.vue]: You need to add keys when iterating over elements.
